project/MicroServiceBuild.scala

```diff
@@ -12,18 +12,15 @@ object MicroServiceBuild extends Build with MicroService {
 private object AppDependencies {
   def apply() = Seq(
     ws,
-    "uk.gov.hmrc"                 %% "microservice-bootstrap" % "10.3.0",
-    "uk.gov.hmrc"                 %% "domain"                 % "5.3.0",
-    "uk.gov.hmrc"                 %% "emailaddress"           % "3.2.0",
+    "uk.gov.hmrc"                 %% "microservice-bootstrap" % "10.9.0",
+    "uk.gov.hmrc"                 %% "domain"                 % "5.6.0-play-25",
+    "uk.gov.hmrc"                 %% "emailaddress"           % "3.3.0",
```

Comment:
emailaddress dependency is now at version 3.4.0, any lower than that will cause a bobby rule violation https://catalogue.tax.service.gov.uk/service/hmrc-email-renderer

Reply:
yeah, we know, we are managing emailaddress library and set it to be a warning, it is not a rule violation/error yet. The reason we don't change it now is that we need to give clients an advanced warning so that their API calls don't start to suddenly throw exceptions were previously were ok.